package.json

```diff
@@ -43,7 +43,7 @@
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-mirage": "~0.2.3",
     "ember-cli-mocha": "^0.11.0",
-    "ember-cli-notifications": "^4.0.8",
+    "ember-cli-notifications": "~4.0.8",
```

Comment:
it appears `4.1.1` or one similar to that is what broke this. Took the easy way out and pinned this instead of ripping it out as I've run out of time to work on this today.